pom.xml

```diff
@@ -163,6 +163,16 @@ under the License.
 			<type>jar</type>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
```

Comment:
Yes,that's right.I have done some work in `flink-runtime` module and `flink-core` before,so i add JMH dependency to the root pom in oder to apply to all modules.I will modify the pom and move the dependency to pom of `flink-benchmark`.